apps/file/mutations.py

```diff
@@ -8,11 +8,9 @@
 
```

Comment:
We don't need this whole code..since everything is done at the dataset level.

Reply:
Yes, thanks for pointing out.